src/gentropy/datasource/gwas_catalog/associations.py

```diff
@@ -1109,7 +1109,7 @@ def from_source(
         """
         return StudyLocusGWASCatalog(
             _df=gwas_associations.withColumn(
-                "studyLocusId", f.monotonically_increasing_id().cast(LongType())
+                "studyLocusId", f.monotonically_increasing_id().cast(StringType())
```

Comment:
This is not a dealbreaker, and has no impact whatsoever: this column is not a "real" `studyLocusId`: this column is temporarily generated to identify original rows of the GWAS Catalog association dataset before explosion. But it is fine.